packages/eslint-config-pwa-kit/index.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+module.exports = {
+    parser: 'babel-eslint',
+    parserOptions: {
+        ecmaVersion: 2017,
+        sourceType: 'module',
+        ecmaFeatures: {
+            jsx: true
+        }
+    },
+    env: {
+        es6: true,
+        node: true,
+        browser: true,
+        jest: true
+    },
+    extends: ['eslint:recommended', 'plugin:react/recommended', 'prettier', 'prettier/react'],
+    plugins: ['header', 'react', 'prettier'],
```

Comment:
```suggestion
    plugins: ['react', 'prettier'],
```
(As per comment in `package.json`)